norddrop/src/norddrop.udl

```diff
@@ -60,6 +60,16 @@ dictionary Config {
 
     /// Limits the number of connection retries afer the `network_refresh()` call.
     u32? connection_retries;
+
+    /// Enable auto retry loop inside libdrop. Conceptually this means that
+    /// libdrop is calling `network_refresh()` automatically with the given
+    /// period in milliseconds. When set to `null` the feature is disabled
+    /// and the application needs to call  `network_refresh()` manually.
+    /// Note the setting `connection_retries` still applies, meaning the retry
+    /// is executed in burst with this number of count.
+    /// For example for a single retry every 5 seconds the application needs to
+    /// set `connection_retries = 1` and `auto_retry_interval_ms = 5000`.
```

Comment:
I left a comment earlier on here to comment it better on what happens when you have `connection_retries = 0` and `auto_retry_interval_ms = 1000` as an example. Could you elaborate on all invariants on both flags?

Reply:
Yeah, sure. Let me do it

Reply:
I've added more examples and focused on `connection_retries` to clarify its behavior.
I hope it's more clear now.